scripts/src/utils/rate_limit.ts

```diff
@@ -1,45 +1,34 @@
 import * as moment from "moment";
 
 import { getRedisClient, RedisAsyncClient } from "../redis";
-import { TooManyRegistrations, TooMuchEarnOrdered } from "../errors";
+import { MarketplaceError, TooManyRegistrations, TooMuchEarnOrdered } from "../errors";
+import { Application } from "../models/applications";
 
 class RateLimit {
 	private readonly bucketPrefix: string;
-	private readonly rateLimitValue: number;
 	private readonly redis: RedisAsyncClient;
 	private readonly windowSize: number = 0;
 	private readonly bucketSize: number = 0;
 	private readonly currentTimestampSeconds: number = 0;
-	private readonly ttl: number = moment.duration({ days: 2 }).asSeconds(); // two days in seconds
+	private readonly ttl: number; // two days in seconds
```

Comment:
i also prefer that the value assignment be done in the ctor, but maybe move the comment as well?